minvws/nl-kat-coordination
# Add start date to report schedule

### Changes
This PR adds the start date input to the Report Schedule section on the Export Setup page in the report flow.
This allows a user to select the start date of the scheduled report.

It also changes the text “This field is required” to “(Required)” because it was not compliant with the designs. The changes are applied to all forms in KAT.

### Issue link
Closes #2779

### Demo
![afbeelding](https://github.com/user-attachments/assets/bd6f9ebf-e52e-4b6a-8ca4-26a1a71aa618)

### QA notes
- Create a new report and select "Yes, repeat"
- Select a date
- After creating a recipe, check if the selected date is visible in the table on the Scheduled Reports page

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
